packages/manager/apps/pci-ai-tools/src/components/order/docker-custom-image/DockerCutomImage.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { z } from 'zod';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { ArrowUpRightFromSquare, Check, Plus } from 'lucide-react';
+import {
+  Button,
+  Card,
+  CardContent,
+  CardHeader,
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+  Input,
+} from '@datatr-ux/uxlib';
+import A from '@/components/links/A.component';
+import ai from '@/types/AI';
+
+interface DockerCustomImageProps {
+  value: string;
+  onChange: (newImage: string) => void;
+  images: ai.job.PresetImage[];
+}
+
+const DockerCustomImageInput = React.forwardRef<
+  HTMLInputElement,
+  DockerCustomImageProps
+>(({ value, onChange, images }, ref) => {
+  const { t } = useTranslation('ai-tools/components/docker-custom-image');
+  const personalImageRules = z
+    .string()
+    .trim()
+    .min(1, {
+      message: t('formDatastoreErrorMinLength', {
+        min: 1,
+      }),
+    });
+
+  const imageSchema = z.object({
+    imageName: personalImageRules,
+  });
+
+  const imageForm = useForm({
+    resolver: zodResolver(imageSchema),
+    defaultValues: {
+      imageName: value,
+    },
+  });
+
+  const onSubmit = imageForm.handleSubmit((formValues) => {
+    onChange(formValues.imageName);
+  });
+
+  return (
+    <Card data-testid="docker-custom-image">
+      <CardHeader>
+        <h5>{t('customImageTitle')}</h5>
+        <div className="pt-4 text-sm">
+          <p>{t('privateImageDesc')}</p>
+          <ul className="list-disc my-3">
+            <div className="ml-8 font-semibold">
+              <li>{t('privateImageDesc1')}</li>
+              <li>{t('privateImageDesc2')}</li>
+              <li>{t('privateImageDesc3')}</li>
+            </div>
+          </ul>
+          <p>
+            {t('privateImageDesc4')}{' '}
+            <A
+              href={
+                'https://docs.ovh.com/gb/en/publiccloud/ai/training/build-use-custom-image/'
+              }
```

Comment:
I think you already have a `getGuideUrl`, it would make sense to reuse it here to ease the transition when we want to move to a locale-based link